AnnoyancesFilter/sections/popups.txt

```diff
@@ -98,6 +98,9 @@ sodapdf.com##.be-tooltip
 !
 ! Specific sites
 !
+manybooks.net#$##drupal-modal { display: none !important; }
+manybooks.net#$#.modal-backdrop { display: none !important; }
+manybooks.net#$#body { overflow: visible !important; padding-right: 0 !important; }
```

Comment:
You break the login window.

Reply:
That's not a big problem ;). But I fixed it anyway.